### DISCUSSION TITLE
How can I use RaspAP with Ubuntu 24.04?

### DISCUSSION THREAD

**Discussion by Lolothepro** [RaspAP/raspap-webgui#1653] [Q&A]
I always have the error:
`Ubuntu Noble Numbat is unsupported. Please install on a supported distro`
Is there a way to bypass this error?

**Comment by billz** [comment]
Use a [compatible operating system](https://docs.raspap.com/#compatible-operating-systems).

**Reply by Lolothepro** [reply]
I can't, I use a WiFi dongle with my Raspberry Pi, only compatible with Linux 6.5+.

**Reply by billz** [reply]
Check again. The latest kernel for RPi OS is 6.6:

```
$ uname -a
Linux rpi5 6.6.31+rpt-rpi-2712 #1 SMP PREEMPT Debian 1:6.6.31-1+rpt1 (2024-05-29) aarch64 GNU/Linux
```
